CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+- Support for `ATmega128A`.
+
 ### Added
+
```

Comment:
Just for the future, in my projects you don't have to update the CHANGELOG as part of your PRs.  I'll take care of this after merging.  Other projects are doing this differently, though - this is usually documented somewhere in the contributing guidelines of each project (if such guidelines exist).

Reply:
Ohw, I will take it into account!